tox.ini

```diff
@@ -0,0 +1,27 @@
+[tox]
```

Comment:
we need a license header here if `.ini` support comment

Reply:
Done